guillotina/contrib/cache/utility.py

```diff
@@ -91,18 +91,23 @@ def get_size(self, value):
         return _default_size
 
     # Set a object from cache
-    async def set(self, key, value, ttl=None):
-        try:
-            size = self.get_size(value)
-            self._memory_cache.set(key, value, size)
-            if ttl is None:
-                ttl = self._settings.get("ttl", 3600)
-            if self._obj_driver is not None:
-                stored_value = serialize.dumps(value)
-                await self._obj_driver.set(CACHE_PREFIX + key, stored_value, expire=ttl)
-            logger.info("set {} in cache".format(key))
-        except Exception:
-            logger.warning("Error setting cache value", exc_info=True)
+    async def set(self, keys, value, ttl=None):
+        if not isinstance(keys, list):
+            keys = [keys]
+        size = self.get_size(value)
+        for key in keys:
+            try:
+                size = self.get_size(value)
```

Comment:
Is this line redundant ? The size is calculated in line 97

Reply:
You're right. Fixed and added test.